apps/web-connect/src/features/checkout/components/ActionSection.tsx

```diff
@@ -64,23 +65,23 @@ export function ActionSection(): JSX.Element {
                 {currency === 'AETH' ? (
                     <PrimaryButton
                         onClick={() => mintWithEth.write?.()}
-                        className={`w-full h-16 ${ready && chain?.id === 42161 ? "bg-[#F30919] global-clip-path" : "bg-gray-400 cursor-default !text-[#726F6F]"} text-lg text-white p-2 uppercase font-bold`}
-                        isDisabled={!ready || chain?.id !== 42161}
+                        className={`w-full h-16 ${ready ? "bg-[#F30919] global-clip-path" : "bg-gray-400 cursor-default !text-[#726F6F]"} text-lg text-white p-2 uppercase font-bold`}
+                        isDisabled={!ready}
```

Comment:
We must not remove the checks for the mainnet chain unless we add them globally somewhere.

This would remove the check for the correct network and could lead to unexpected behaviour

Reply:
I believe these are all cleaned up now. Thank you for the guidance!